_assets/js/youtubeplayer.js

```diff
@@ -171,6 +181,12 @@
     }
 
     function closeVideo(options) {
+      // Bail if yt_player is falsey or required
+      // methods not present.
+      if (!yt_player || !yt_player.stopVideo) {
+        return;
```

Comment:
Typically I like to see a warning logged to the console for early returns like this, especially in untyped code as it could easily get out of date after a refactoring.